client/js/analysis.js

```diff
@@ -19,38 +19,83 @@ async function fetchResponse() {
 /**
  * Handles populating all elements on the screen
  *
- * @param {object} json the json object
+ * @param {object} json the json object from backend
  */
 function populationHandler(json) {
   const charts = document.getElementById('charts');
   const list = document.getElementById('list');
+  const card = document.getElementById('videocard-wrapper');
 
   removeAllChildNodes(charts);
   removeAllChildNodes(list);
+  removeAllChildNodes(card);
   populateDonutCharts(json);
+  showCommentHeader();
   const time = populateComments(json);
   determineOverall(json, time*1500);
 }
 
 /**
  * Puts the charts on the screen
  *
- * @param {object} json the json object
+ * @param {object} json the json object from backend
  */
 function populateDonutCharts(json) {
   const map = json.perspectiveMap;
   for (const key in map) {
-    if (key) { // this is here bc of the linter
+    if (Object.prototype.hasOwnProperty.call(map, key)) {
       // Convert to string for type compatibility
-      addDonutChart(key, String(100 - map[key]*100));
+      addDonutChart(key, (100 - map[key]*100).toString());
     }
   }
 }
 
+/**
+ * Creates the card that displays on a result match
+ *
+ * @param {object} json the json object from backend
+ */
+function createCard(json) {
+  const id = json.videoId;
+  const name = json.videoInfo.name;
+  const channel = json.videoInfo.channel;
+
+  const el = document.getElementById('videocard-wrapper');
+
+  const card = document.createElement('div');
+  card.setAttribute('id', 'videocard');
+
+  const img = document.createElement('img');
+  img.classList = 'card-image';
+  img.setAttribute('src', `https://img.youtube.com/vi/${id}/sddefault.jpg`);
+  card.appendChild(img);
+
+  const videoInfo = document.createElement('div');
```

Comment:
can you move this line 73 to right before line 84. That way I'm able to keep track of what's happening to `videoInfo` in one block